experimental/packages/opentelemetry-instrumentation/src/instrumentationNodeModuleDefinition.ts

```diff
@@ -22,7 +22,7 @@ import {
 export class InstrumentationNodeModuleDefinition<T>
```

Comment:
@dyladan I seem to remember that we discussed changing this to `<T=any>` at some point so that it can be left out for reasons like this. WDYT?